src/Application.kt

```diff
@@ -59,9 +111,10 @@ fun main() {
 
     val server = embeddedServer(Netty, port = port) {
         Application(
-            userRepository,
-            twilioChatClient,
-            this
+            userRepository = userRepository,
```

Comment:
Seguindo o mesmo padrão utilizado do `sessionStorage`:

```suggestion
            userRepository = InMemoryUserRepository(),
```